csharp_dotnetcore/81.Facebook-Events/FacebookEventsBot.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Facebook_Events_Bot.FacebookModel;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.AI.Luis;
+using Microsoft.Bot.Builder.AI.QnA;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Choices;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Facebook_Events_Bot
+{
```

Comment:
Use our standard namespace. Microsoft.BotBuilderSamples

Reply:
@cleemullins I actually see a lot of samples with namespaces like this. This should be one of the things to do for the final pass on samples.

What is the correct namespace out of the following 2:

```cs
namespace Microsoft.BotBuilderSamples.Facebook_Events_Bot
```
or

```cs
namespace Microsoft.BotBuilderSamples
```